src/OpenSage.Game.Tests/Data/Ini/IniFileTests.cs

```diff
@@ -19,37 +19,45 @@ public IniFileTests(ITestOutputHelper output)
         [Fact]
         public void CanReadIniFiles()
         {
-            // TODO: Finish INI parsing for BFME and subsequent games.
+            // TODO: Finish INI parsing for BFME2 and subsequent games.
             var gameDefinitions = new[]
             {
                 GameDefinition.FromGame(SageGame.CncGenerals),
-                GameDefinition.FromGame(SageGame.CncGeneralsZeroHour)
+                GameDefinition.FromGame(SageGame.CncGeneralsZeroHour),
+                GameDefinition.FromGame(SageGame.Bfme),
+                //GameDefinition.FromGame(SageGame.Bfme2),
             };
 
-            InstalledFilesTestData.ReadFiles(".ini", _output, gameDefinitions, entry =>
+            foreach(var gameDefinition in gameDefinitions)
```

Comment:
Why this change?

Reply:
this is to get the right game definition here:
var dataContext = new IniDataContext(entry.FileSystem, gameDefinition.Game);
the old approach was to use always SageGame.Generals, but since things like animations are game dependant this wouldnt work, so we need the proper game definition here.